systemd/emulation.go

```diff
@@ -124,20 +124,21 @@ func (s *emulation) LogReader(services []string, n int, follow, namespaces bool)
 	return nil, fmt.Errorf("LogReader")
 }
 
-func (s *emulation) EnsureMountUnitFile(description, what, where, fstype string) (string, error) {
+func (s *emulation) EnsureMountUnitFile(description, what, where, fstype string, flags *EnsureMountUnitFlags) (string, error) {
```

Comment:
Passing EnsureMountUnitFlags by value is probably better here. It's less worrying about escape analysis to optimize this away, less chance of a nil pointer.

Reply:
Changed now